runtime/westend/src/weights/xcm/pallet_xcm_benchmarks_fungible.rs

```diff
@@ -65,19 +64,23 @@ impl<T: frame_system::Config> WeightInfo<T> {
 	// Storage: Dmp DownwardMessageQueueHeads (r:1 w:1)
 	// Storage: Dmp DownwardMessageQueues (r:1 w:1)
 	pub(crate) fn transfer_reserve_asset() -> Weight {
-		(99_639_000 as Weight)
+		(54_395_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(7 as Weight))
 			.saturating_add(T::DbWeight::get().writes(5 as Weight))
 	}
+	// Storage: Benchmark Override (r:0 w:0)
+	pub(crate) fn reserve_asset_deposited() -> Weight {
+		(2_000_000_000_000 as Weight)
```

Comment:
Actually no, because Westmint doesn't have a configured reserve location, so it can't be benched, and as such, any incoming `ReserveAssetDeposited` should have a high cost as it can't really be executed.